### PR TITLE
[SERIOUS] Improves admin tooling for radiation

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Adds a new buildmode for cleaning radiation
- left click to remove radiation on a target
- ctrl-left to lightly clean the entire screen
- right click - deep clean a turf (everything on that turf will loose all radiation)
- ctrl-right click - deep clean entire screen (remove all radiation in sight, including in people's worn bags, etc)

Also adds an admin verb that deletes ALL radiation components, if thats needed. Locked behind R_DEBUG.

## Why It's Good For The Game

Admins requested it

## Images of changes

image
image

## Testing

See above

## Changelog
NPFC

### CODE REVIEW

**Comment by 1080pCat** [comment]
> * ctrl-left click to "deep clean", removing all radiation in that turf, from all its contents
> 
> * right click - deep clean entire screen

i'd personally suggest this be the other way around keybind wise, just to the one that could cause more potential lag be harder to miss input

File: code/modules/buildmode/submodes/antirad_mode.dm [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/datum/buildmode_mode/clean_radiation
+	key = "rad"
+
+/datum/buildmode_mode/clean_radiation/show_help(mob/user)
+	to_chat(user, "<span class='notice'>***********************************************************</span>")
+	to_chat(user, "<span class='notice'>Left Mouse Button on obj/turf/mob = Remove radiation of selected target.</span>")
+	to_chat(user, "<span class='notice'>Ctrl-Left Mouse Button on screen = Remove all radiation in your sight.</span>")
+	to_chat(user, "<span class='notice'>Right Mouse Button on obj/turf/mob = Deep clean: Remove radiation of all recursive contents of turf.</span>")
+	to_chat(user, "<span class='notice'>Ctrl-Right Mouse Button on screen = Deep clean: Remove radiation of all recursive contents in your sight.</span>")
+	to_chat(user, "<span class='notice'>***********************************************************</span>")
```
Comment (DGamerL) [inline code/modules/buildmode/submodes/antirad_mode.dm:10]: Can you make these into a list and then `.Join()` them?

**Review by DGamerL — COMMENTED** [review]
LGTM otherwise

File: code/modules/buildmode/submodes/antirad_mode.dm (hunk 1 above)
Comment (Contrabang) [reply code/modules/buildmode/submodes/antirad_mode.dm:10]: I was considering that, but then I feel like I would need to refactor ALL of them to use chat boxes and message lists, which I don't really want to do for a PR of this size

**Review by DGamerL — APPROVED** [review]
LGTM, I'd like a second reviewer to look at this aswell

**Review by Burzah — APPROVED** [review]

**Pull request merged**